.github/workflows/pull-request.yaml

```diff
@@ -51,6 +51,17 @@ jobs:
         with:
           sarif_file: 'trivy-results.sarif'
 
+  cargo-audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - run: rustup toolchain install stable --profile minimal
+      - uses: rustsec/audit-check@69366f33c96575abad1ee0dba8212ae2a06c12a9 # v2.0.0
```

Comment:
Based on the official [release](https://github.com/rustsec/audit-check/releases/tag/v2.0.0) the pinned commit is slightly wrong, no?
```suggestion
      - uses: rustsec/audit-check@69366f33c96575abad1ee0dba8212993eecbe998 # v2.0.0
```